AD-User-README.md

```diff
@@ -0,0 +1,36 @@
+### [AD User Credentials]
+
+[Azure AD/Entra ID](https://www.microsoft.com/en-gb/security/business/identity-access/microsoft-entra-id) based authentication allows for end users to request short lived OAuth tokens from Azure, to be used for authentication to downstream services integrated with AD
+
+AD Authentication is an Industry standard method, using the OAuth standard to secure access to sensitive Data. This also allows for user level audits for data access, providing a complete audit trail for sensitive data as well as centrally managed RBAC
+
+**Why should you use AD User Credentials?**
+
+- Your Enterprise Policy forbids the use of long lived credentials (Username/Password, Access Key, SAS token)
+- You require strong auditing of all data reads & writes
+- You would like to use OAuth standards to propagate short lived credentials to Domino Workloads
+- Your Data source access is centrally managed via AD and you want to propogate the same access to Domino
+
+**AD User Credentials work as follows:**
+
+1. Each Domino instance establishes an OIDC trust with an AD Application. This allows Domino to request and manage tokens on the user's behalf
```

Comment:
For each of these steps it would be easier for the user to have link from the setup guide with the screen shots and instructions.